api_spec.yaml

```diff
@@ -890,7 +890,8 @@ paths:
         - OAuth2 Recipe
       operationId: updateOAuth2Client
       description: |
-        Update an OAuth2 client
+        Update an OAuth2 client, 
+        [Click here](https://app.code2flow.com/nXFGV7pIFOFl) for flow diagram
```

Comment:
- in the input validation, you have "name is not && (name...". What do you mean by "name is not &&"?
- You don't need the `getOauth2ClientById_transaction` function part